src/pages/registration/FileList.tsx

```diff
@@ -99,36 +95,56 @@ export const FileList = ({ title, files, uppy, remove, baseFieldName }: FileList
                     <Box sx={{ display: 'flex', gap: '0.5rem', alignItems: 'center' }}>
                       {t('registration.files_and_license.license')}
                       <HelperTextModal
+                        maxWidth="xl"
                         modalTitle={t('registration.files_and_license.licenses')}
                         modalDataTestId={dataTestId.registrationWizard.files.licenseModal}
                         buttonDataTestId={dataTestId.registrationWizard.files.licenseHelpButton}>
-                        <Typography sx={{ mb: '1rem' }}>
-                          {t('registration.files_and_license.file_and_license_info')}
-                        </Typography>
-                        {activeLicenses.map((license) => (
-                          <Box key={license.id} sx={{ mb: '1rem', whiteSpace: 'pre-wrap' }}>
+                        <Trans
+                          i18nKey="registration.files_and_license.file_and_license_info"
+                          components={{
+                            p: <Typography gutterBottom />,
+                            ccLink: <OpenInNewLink href="https://creativecommons.org/licenses/" />,
+                          }}
+                        />
+                        <Divider sx={{ my: '1rem', borderColor: 'black', borderWidth: '1px' }} />
+                        <Box
+                          sx={{
+                            display: 'grid',
+                            gridTemplateColumns: { md: '1fr', lg: 'repeat(3, 1fr)' },
+                            gap: '1.5rem',
+                          }}>
+                          {activeLicenses.map((license) => (
+                            <Box key={license.id}>
```

Comment:
Om dette bare er for gruppering klarer vi oss uten MUI-avhengighet
```suggestion
                            <div key={license.id}>
```